### ISSUE TITLE
Weird question about the loss and mAP?

### ISSUE THREAD

**Issue by tdzdog** [open-mmlab/OpenPCDet#882]
Recently, I have made some modifications to the PV-RCNN++ model on Waymo but I met a very weird problem. The training loss is OK, not different from my previous experiments. But the final validation mAP is very awful. Only 50mAP. Then I tried to test on the train set. The result is still very bad. I noticed a severe tiny box problem. I am totally confused right now.  I have tried the codes many times and it all worked well. Are there potential flaws in the current loss?
